scripts/bmm_examples/Example1_Zhang&Luck2008.R

```diff
@@ -237,13 +245,25 @@ ggsave(
 ## 4.4) Test hypothesis --------------------------------------------------------
 # specify hypothesis
 hyp_ZL2008 <- c(
-  hyp_kappa_1v2 = "kappa1_setsize1 = kappa1_setsize2",
-  hyp_kappa_2v3 = "kappa1_setsize2 = kappa1_setsize3",
-  hyp_kappa_3v6 = "kappa1_setsize3 = kappa1_setsize6",
-  hyp_theta_1v2 = "theta1_setsize1 = theta1_setsize2",
-  hyp_theta_2v3 = "theta1_setsize2 = theta1_setsize3",
-  hyp_theta_3v6 = "theta1_setsize3 = theta1_setsize6"
+  hyp_kappa_1v2 = "kappa_setsize1 = kappa_setsize2",
+  hyp_kappa_2v3 = "kappa_setsize2 = kappa_setsize3",
+  hyp_kappa_3v6 = "kappa_setsize3 = kappa_setsize6",
+  hyp_theta_1v2 = "thetat_setsize1 = thetat_setsize2",
+  hyp_theta_2v3 = "thetat_setsize2 = thetat_setsize3",
+  hyp_theta_3v6 = "thetat_setsize3 = thetat_setsize6"
 )
 
 # test hypothesis
-hypothesis(fit_ZL_mixModel,hyp_ZL2008)
+hypothesis(ZL_fit,hyp_ZL2008)
+plot(hypothesis(ZL_fit,hyp_ZL2008))
+
+# 5) Extract brms info from bmm object -----------------
+ZL_mockfit <- bmm(
```

Comment:
why do we need a mockfit? the formula should be available from the bmmfit object